back/db/migrate/20230606132255_add_voting_method_to_participation_context.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class AddVotingMethodToParticipationContext < ActiveRecord::Migration[6.1]
+  def change
+    %i[projects phases].each do |tablename|
+      add_column tablename, :voting_method, :string
+      execute "UPDATE #{tablename} SET voting_method = 'budgeting' WHERE participation_method = 'budgeting'"
+      execute "UPDATE #{tablename} SET participation_method = 'voting' WHERE participation_method = 'budgeting'"
+
+      add_column tablename, :voting_max_votes_per_idea, :integer
+      rename_column tablename, :max_budget, :voting_max_total
+      rename_column tablename, :min_budget, :voting_min_total
+      add_column tablename, :voting_term, :jsonb, default: {}
```

Comment:
Haven't seen any specific tests for the two new columns, but I guess they will probably come in later iterations as they are not used at the moment